setup.py

```diff
@@ -12,12 +12,14 @@
     description='Data feed components for Cyber Hygiene',
     #long_description=open('README.txt').read(),
     install_requires=[
-        "cyhy-core >= 0.0.2",
+        "pymongo >= 2.9.2, < 3",
```

Comment:
The list of requirements should be alphabetized.

Reply:
(I realize that they weren't alphabetized before and you had no idea they should have been.  I just figured it's something we may as well fix now since it slipped through the cracks before.